selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/CheckWorkingWithTabsByUsingContextMenuTest.java

```diff
@@ -96,16 +96,6 @@ public void checkWorkingWithTabsByUsingContextMenu() {
     editor.waitTabIsNotPresent(NAME_TO_HTML);
     editor.closeAllTabs();
 
-    // close other tabs by using context menu
```

Comment:
Do you know what was the propose to repeat the same verification twice?

Reply:
No, I don't, but it hasn't any explanation, so I have decided that this is duplicated code and removed it.

Reply:
Okay.